File: Practice/src/introduction_coding_tests/RotateArray.py
def solution(numbers, direction):
    if direction == "left":
        numbers.append(numbers.pop(0))
    else:
        numbers.insert(0, numbers.pop())
    return numbers
    # return [numbers[-1]] + numbers[:-1] if direction == 'right' else numbers[1:] + [numbers[0]]

File: Practice/src/introduction_coding_tests/RotateArray.java
package introduction_coding_tests;

import java.util.stream.IntStream;

public class RotateArray {
//	배열 회전시키기
	public int[] solution(int[] numbers, String direction) {
		int n = numbers.length;
		return IntStream.range(0, numbers.length)
				.map(i -> direction.equals("left") ? numbers[(2 * n + i + 1) % n] : numbers[(2 * n + i - 1) % n])
				.toArray();
	}
}